packages/web-app/src/reducers/GCReducer.js

```diff
@@ -68,6 +68,7 @@ import updateMassif from './UpdateMassifReducer';
 import updateName from './UpdateNameReducer';
 import updateOrganization from './UpdateOrganization';
 import updatePerson from './UpdatePerson';
+import getSnapshots from './GetSnapshotReducer';
```

Comment:
`import snapshot from ./SnapshotReducer`

When contributing too a large existing project, observe what has been done previously, especially about naming conventions 😉 

Also, put it in alphabetical orders (same : look at what is already here).

Reply:
As long as we see `Update` and `Create`, we thought that `Get` was needed ...
We will change this !